docs/source/api/python/dgl.distributed.rst

```diff
@@ -29,7 +29,7 @@ Distributed Node Embedding
 ---------------------
 .. currentmodule:: dgl.distributed.nn.pytorch
```

Comment:
is the name space correct?

Reply:
It is actually under dgl.distributed.nn.pytorch